src/routes/health.ts

```diff
@@ -0,0 +1,14 @@
+import { Router, Request, Response } from 'express';
+import mongoose from 'mongoose';
+import HealthCheckController from '@/controllers/HealthCheckController';
```

Comment:
imo should be @controllers not @/controllers

Reply:
Both ways doesn't work.